### PR TITLE
Cloudraid merge2

### PR DESCRIPTION
**Pull request by mattw-mega**
Initial implementation of CloudRAID(tm) client code in the SDK

### CODE REVIEW

**Comment by mattw-mega** [comment]
trigger compilation

**Comment by mattw-mega** [comment]
trigger compilation

File: examples/megacli.cpp [hunk 1]
```diff
@@ -2074,6 +2074,253 @@ static void store_line(char* l)
     line = l;
 }
 
+#if __cplusplus >= 201100L
+
+class FileFindCommand : public Command
```
Comment (sergiohs84) [inline examples/megacli.cpp]: I'm aware this is not the only declaration added directly to the .cpp, but I'd prefer to keep declaration in headers and definitions in sources. Not a biggie :)

**Comment by sergiohs84** [comment]
Closing this PR, replaced by another with the standard naming: https://github.com/meganz/sdk/pull/1370

**Pull request closed without merging**